notebooks/stable-fast-3d/stable-fast-3d.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import math
</pre><p>can you please consider to move conversion and inference wrappers to file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Moved</p>